macros/edr/dbt_artifacts/upload_dbt_invocation.sql

```diff
@@ -102,6 +106,16 @@
         {% do return(invocation_args_dict.selector) %}
     {% elif invocation_args_dict and invocation_args_dict.selector_name %}
         {% do return(invocation_args_dict.selector_name) %}
+    {% elif invocation_args_dict and invocation_args_dict.INVOCATION_COMMAND %}
+        {% set match = modules.re.search(
```

Comment:
selector is not available in fusion, so getting it with a regex from the command